### DISCUSSION TITLE
prevent content copying

### DISCUSSION THREAD

**Discussion by tkatsageorgis** [sumatrapdfreader/sumatrapdf#2719] [General]
Hello there, 
I am a big fan of Sumatra PDF reader and I use it for many years. I create pdf files using libre (v.7.0.3.1). Just before exporting the pdf, I chose the security tab and type a password for changes and uncheck the 'Enable Content Copying' choice. After saving and opening the newly created file using Sumatra 3.1.2, and trying to copy a part of the pdf file I get the message: 'Copying text was denied'. 
If I use version 3.3.3 or the latest one 3.4.6, I don't get the prevention message. Any ideas? 
Thanks in advance,
Thanassis

**Comment by GitHubRulesOK** [comment]
3.1.2 did not need to edit the file thus respected "no copy as text"
3.3 on needs to add annotation but has to use the decryption password for changes to appearance so by default ignore all preferences to annotate / highlight etc. is allowed but should preserve master password to save.

The only function a password on a .Zip .DocX or PDF serves is to slow down opening and closing content for decryption to view, once you give the new owner your user password they are the new owner, and can reset the password but most new owners just strip it. Adobes own PDF products will of course work different to all others.
